crates/opencascade-sys/src/lib.rs

```diff
@@ -534,5 +554,33 @@ pub mod ffi {
 
         pub fn Shape(self: &BRepMesh_IncrementalMesh) -> &TopoDS_Shape;
         pub fn IsDone(self: &BRepMesh_IncrementalMesh) -> bool;
+
+        type TopLoc_Location;
+        #[cxx_name = "construct_unique"]
+        pub fn TopLoc_Location_ctor() -> UniquePtr<TopLoc_Location>;
+
+        type Handle_Poly_Triangulation;
+        pub fn IsNull(self: &Handle_Poly_Triangulation) -> bool;
```

Comment:
This can probably be removed unless you think we'll be using it.

Reply:
I feel like leaving it is better. Technically there could be many reasons that the request fails but using IsNull is clear enough.